website/docs/commands/repl.md

```diff
@@ -80,6 +80,30 @@ scala> :quit
 
 </ChainedSnippets>
 
+## Using Toolkit in REPL
+It is also possible to start the scala-cli REPL with [toolkit](https://scala-cli.virtuslab.org/docs/guides/introduction/toolkit/) enabled
+
+<ChainedSnippets>
+    
+```bash ignore
+scala-cli repl --toolkit default
+```
+
+```text
+Welcome to Scala 3.3.1 (17, Java OpenJDK 64-Bit Server VM).
+Type in expressions for evaluation. Or try :help.
+                                                                                
+scala> import os._
+                                                                                
+scala> os.pwd
+val res0: os.Path = /Users/yadukrishnan/test
+scala> :quit
+
+```
+</ChainedSnippets>
+
+Since we started the repl with toolkit enbled, we can use the libraries included in the toolkit direclty. In the above example, `os-lib` library from the toolkit is used to print the current path. 
```

Comment:
```suggestion
Since we started the repl with toolkit enabled, we can use the libraries included in the toolkit directly. In the above example, the `os-lib` library from the toolkit is used to print the current path. 
```